### PR TITLE
[CountriesResponse] country code added (using iso2 format)

### PR DESCRIPTION
**Pull request by buster95**
Country info:
    - [x] latitude,
    - [x] longitude,
    - [x] iso2 name format,
    - [x] iso3 name format,
    - [x] flag url

### CODE REVIEW

**Comment by RyanHarlow** [comment]
the require statement has the file name wrong
var countryUtils = require('./../utils/countryUtils');
should be
var countryUtils = require('../utils/country_utils.js');

**Comment by RyanHarlow** [comment]
also some of the countries aren't working if you try gb country code for the united kingdom it maps it to United Kingdom whereas the api will only accept uk on the countries endpoint

**Comment by ebwinters** [comment]
https://cdn.discordapp.com/attachments/689538810593542161/691411943881441339/Screen_Recording_2020-03-22_at_5.23.16_PM.mov
I'm currently working on this, see the preview. Pls wait to merge this until I get my PR in so you know what country names we are using and can plan code accordingly

**Comment by buster95** [comment]
I sorry friend 😅, was fixed 😎

**Comment by ebwinters** [comment]
@buster95 no it's not fixed because I haven't pushed the standardized country names. For example, czechia is now czech republic. So if you push you code, it will not work with the API for certain countries.

**Comment by buster95** [comment]
> @buster95 no it's not fixed because I haven't pushed the standardized country names. For example, czechia is now czech republic. So if you push you code, it will not work with the API for certain countries.

image

**Comment by ebwinters** [comment]
you're not understanding. Czechia is an accepted name, but it maps to czech republic in the API. That is the standardized name since that is what most people know.If someone queries czechia though, it will work but behind the scenes will change to czech repub

**Comment by buster95** [comment]
> https://cdn.discordapp.com/attachments/689538810593542161/691411943881441339/Screen_Recording_2020-03-22_at_5.23.16_PM.mov
> I'm currently working on this, see the preview. Pls wait to merge this until I get my PR in so you know what country names we are using and can plan code accordingly

Ok if you are working in this, can you ensure to add country code field in response
image

**Comment by buster95** [comment]
I would like to add country flag field

**Comment by ebwinters** [comment]
you can make a pr for that

**Comment by buster95** [comment]
> you're not understanding. Czechia is an accepted name, but it maps to czech republic in the API. That is the standardized name since that is what most people know.If someone queries czechia though, it will work but behind the scenes will change to czech repub

But I didn't work in that, I was working on add country code field

**Comment by ebwinters** [comment]
then make an issue to add country code to response and someone will do it

**Comment by buster95** [comment]
> then make an issue to add country code to response and someone will do it

@ebwinters this is a PR for add country code in response

**Comment by alessandrojcm** [comment]
Hello here! Is this clear to merge? I really need it ASAP.

**Comment by buster95** [comment]
image

File: utils/country_utils.js [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+    return 'NOT DATA';
+}
+
+const getCountryName = function (countryCode) {
+    for (const index in isoCountries) {
+        if (isoCountries[index].ccode.toLowerCase() == countryCode.toLowerCase()) {
+            return isoCountries[index].cname;
+        }
+    }
+    return 'NOT DATA';
```
Comment (ebwinters) [inline utils/country_utils.js]: should say 'no data'

File: utils/country_utils.js [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+    { 'ccode': 'ZW', 'cname': 'Zimbabwe' }
+];
+
+const getCountryCode = function (countryName) {
+    for (const index in isoCountries) {
+        if (isoCountries[index].cname.toLowerCase() == countryName.toLowerCase()) {
+            return isoCountries[index].ccode;
+        }
+    }
+    return 'NOT DATA';
```
Comment (ebwinters) [inline utils/country_utils.js]: should say no data

**Comment by buster95** [comment]
you can check dude 🖖

**Comment by EliteDaMyth** [comment]
Seems fine will merge.

**Comment by disruptivne** [comment]
Here is the whole dataset for country analysis:

To match country name to country code. I used it for merging JHU CSSE and Worldmeters dataset :)

https://coronadatasource.org/open-data/geo-data/country-names-to-country-codes-translation/

**Comment by disruptivne** [comment]
Also exceptions in country code
image

**Comment by disruptivne** [comment]
If you need any help with countries dataset and Geo, feel free to ask :)

**Comment by EliteDaMyth** [comment]
@all-contributors please add @buster95 for code

**Pull request merged**